### ISSUE TITLE
Delete the H1 of the header

### ISSUE THREAD

**Issue by frank699** [Nikeo/customizr#1431]
Hello,
I would like to delete the H1 of Customizr Theme that is on every webpage of my website in the Header. It show that in the HTML client : 

`<div class="brand span3 pull-left ">
        <h1><a class="site-title" href="https://www.monwebsite.com/" title="My website| ">My website</a></h1>
      `</div>`

Could someone tell me how can I remove the H1 please ?
Thank you :)

**Comment by chlipalski** [comment]
Hello,
Customizr 4.1.20, the new installation.
I have 2 H1 ????

```
<h1 class="navbar-brand col-auto " >
--
  | <a class="navbar-brand-sitename " href="https://waldemar.chlipalski.pl/"  title="My Blog \| My WordPress Blog">
  | <span>My Blog</span>
  | </a>
  | </h1>
```

Could someone tell me how can I remove the H1 please ?
Thank you :)

**Comment by ghost** [comment]
fixed in #1760
